op-mode-definitions/containers.xml.in

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="add">
```

Comment:
Why do I need to add a container image from op-mode? I would assume when I create a container and link it to the image it gets automatically pulled.

Reply:
The images are automatically pulled in commit.
But you can also pull images from op-mode. It’s only an option for op-mode.

Reply:
So this means "pull in an image now yet referenced for future use"? Sounds valid for remote deployments.

How does one update an image? Jenkins uses a flag named `alwaysPull true` to indicate an image should always be pulled from the registry first (and checked for updates) before launching the container.

Reply:
As I understand, When you deploy a new container, and the image doesn't exist in the "local registry" it pulled it from the external registry.
If an image exists in the local registry, it's using this image. I think for update you need pull image again by "sudo  podman image pull xxxx" I don't use any such flags.

Reply:
We can work that onenout later, but should keep it in the backlog.